app/views/organisms/new.html.erb

```diff
@@ -0,0 +1,11 @@
+<%= render 'shared/navbar' %>
+
+<div class="container">
+<%= simple_form_for(@organism) do |f| %>
+  <%= f.input :name %>
+  <%= f.input :body_temperature %>
+  <%= f.input :age %>
+  <%= f.input :species, :collection => %w[human pet plant], prompt: "-- SELECT ONE --" %>
```

Comment:
here it should be included an address input tab. Then the geocode gem automatically gets latitude and longitude for the map render